### ISSUE TITLE
misspelled option "enableKeyboardAccessibility"

### ISSUE THREAD

**Issue by ruanmed** [zefoy/ngx-ace-wrapper#76]
The `enableKeyboardAccessibility` option cannot be used when using ace via the wrapper in Angular application.

```ts
    this.editor.setOption('enableKeyboardAccessibility', true);
    // or
    // this.editor.setOptions({ enableKeyboardAccessibility: true });
```

Produces the following warning the console:

```
misspelled option "enableKeyboardAccessibility"
```

The project seems to have ace-builds as a dependency, which had `enableKeyboardAccessibility` option implemented at [version 1.17.0 of ace](https://github.com/ajaxorg/ace/issues/5114), but it seems actually only [brace](https://github.com/thlorenz/brace) is used under the hoods, which does not receive updates since ace 1.2.9.

Not sure why ace-builds is even a dependency of this package.
